packages/completer/src/handler.ts

```diff
@@ -306,12 +300,16 @@ export class CompletionHandler implements IDisposable {
       return;
     }
 
-    // If the part of the line before the cursor is white space, return.
-    if (line.slice(0, position.column).match(/^\s*$/)) {
-      this._enabled = false;
-      model.reset(true);
-      host.classList.remove(COMPLETER_ENABLED_CLASS);
-      return;
+    // If no line or cursor has not characters before
```

Comment:
Highlighting grammar issue here

Reply:
I updated the language to be more clear here.